src/features/wisdom/wisdomOAuthProvider.ts

```diff
@@ -500,4 +492,10 @@ export class WisdomAuthenticationProvider
       }
     }
   }
+
+  /* Get base uri in a correct formatted way */
+  public getBaseUri() {
```

Comment:
The function can be moved to utils folder `wisdom.ts` with `settingsManager` as argument

Reply:
updated